src/core/formatters.ts

```diff
@@ -131,6 +131,56 @@ export module Formatters {
   }
 
   /**
+   * Creates a formatter for values that displays abbreviated values
+   * and uses standard short scale suffixes (thousands, millions, billions, trillions,
+   * quadrillions).
+   *
+   * Numbers with a magnitude outside of (10 ^ (-precision), 10 ^ (15 + precision)) are shown using
+   * scientific notation to avoid creating extremely long decimal strings.  The inputs to the
+   * formatter are primarily expected to be ranges of common currency values.
+   *
+   * @param {number} [precision] the number of decimal places to show (default 3)
+   *
+   * @returns {Formatter} A formatter for large numbers.
+   */
+  export function shortScale(precision = 3) {
+    verifyPrecision(precision);
+    var suffixes = "KMBTQ";
+    var eFormatter = d3.format("." + precision + "e");
+    var fFormatter = d3.format("." + precision + "f");
```

Comment:
Also instead of using `d3`'s `fixed` formatter, you could use our own formatter, not sure which one is better.